src/functions-and-arrays.js

```diff
@@ -52,16 +110,32 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
+function uniquifyArray(wordsUnique) {
```

Comment:
I wouldn't call this variable wordsUnique, because it is the array which contains also duplicate entries.
Better would be something like "wordsWithDuplicateEntries"